Resources/public/js/views/services/plugins/ez-savedraftplugin.js

```diff
@@ -106,6 +108,7 @@ YUI.add('ez-savedraftplugin', function (Y) {
             }
 
             if ( error ) {
+                serverSideErrorCallback(this._getServerSideFieldsErrors(response.document.ErrorMessage.errorDetails.fields));
```

Comment:
that's not super readable and the method responsibilities are not very clear. I would write that in the following way:
```js
this._parseServerFieldsErrors(response, serverSideErrorCallback);
```

so that `_parseServerFieldsErrors` is fully responsible for parsing the response in case of error (at the moment, this is done for a part here and for a part in `_getServerSideFieldsErrors`.